src/test/scala/com/microsoft/hyperspace/index/IndexLogManagerImplTest.scala

```diff
@@ -187,7 +187,7 @@ class IndexLogManagerImplTest
   }
 
   test("testUpdateLatestStableLog fails with exception if unable to find a valid log entry") {
-    val path = new Path(testRoot, UUID.randomUUID().toString)
+    val path = PathUtils.makeAbsolute(new Path(testRoot, UUID.randomUUID().toString))
```

Comment:
can we just make `testRoot` an absolute path?

Reply:
thanks @imback82 , done